### DISCUSSION TITLE
[Feature Request] Support for Vue 3.0？

### DISCUSSION THREAD

**Discussion by ZhQuella** [Happy-Coding-Clans/vue-easytable#508] [General]
### 选择要提交 issue 的库
vue-easytable

### Issue 类型
Feature

### Issue 标题
Support for Vue 3.0？

### 这个功能解决了什么问题？
Is Vue3.0 available?

### 你期望的 API 是什么样的？
Hope to support Vue3.0.

**Comment by huangshuwei** [comment]
Thanks for your report. I've put this feature on the todo list:
https://github.com/Happy-Coding-Clans/vue-easytable/projects/3#card-74542580

**Reply by adoin** [reply]
> Thanks for your report. I've put this feature on the todo list: https://github.com/Happy-Coding-Clans/vue-easytable/projects/3#card-74542580

好呀好呀

**Comment by kekswar** [comment]
Some news? 

Are looking forward to!

**Comment by huangshuwei** [comment]
Thank you for your expectations. At present, it is mainly my part-time maintenance. Due to my limited energy, I will continue to develop new functions based on vue2. It is estimated that the development of vue3 will not start until the new features(column resize、row sort etc.) based on vue2 are developed and improved.

**Comment by luisdoniad** [comment]
duplicate of #391

**Comment by Dwhyte** [comment]
Hello, I'm just checking to see if there are any updates on the Vue 3 port. :) Thanks for all your hard work!

**Comment by GenericArt** [comment]
Is this still in the works or has it been dropped?

**Comment by yyang0611** [comment]
Hi, thanks again for maintaining vue-easytable.

Our team is still using vue-easytable heavily in a production project, and we rely on several spreadsheet-like features such as:
- editable cells
- clipboard operations
- autofill
- context menu
- column resize / hidden columns
- footer summary

We are planning to migrate part of our frontend to Vue 3 + TypeScript, and vue-easytable is one of the biggest blockers because these interactions are hard to replace with ordinary table components.

I saw your earlier note that Vue 3 support would come after the Vue 2 feature work. I wanted to check whether there are any newer updates on the Vue 3 port or roadmap.

Even an experimental or partial Vue 3 compatible version would be very helpful for teams like us. If needed, we would also be willing to help test.

Thanks.
